docs/admin/addons.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+
+This page lists some of the available add-ons and links to their respective installation instructions.
+
+## Networking and Network Policy
```

Comment:
I've toned down the 'marketing speak' in https://github.com/kubernetes/kubernetes.github.io/pull/1265/commits/b2458c467b6f1b65e689912600502b1d0f6d89b8 😄